sdk/eventhub/Azure.Messaging.EventHubs/src/Consumer/LastEnqueuedEventProperties.cs

```diff
@@ -68,5 +70,89 @@ public struct LastEnqueuedEventProperties
                  sourceEvent?.LastPartitionPropertiesRetrievalTime)
         {
         }
+
+        /// <summary>
+        ///   Determines whether the specified <see cref="EventPosition" /> is equal to this instance.
+        /// </summary>
+        ///
+        /// <param name="other">The <see cref="EventPosition" /> to compare with this instance.</param>
+        ///
+        /// <returns><c>true</c> if the specified <see cref="EventPosition" /> is equal to this instance; otherwise, <c>false</c>.</returns>
+        ///
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool Equals(LastEnqueuedEventProperties other)
+        {
+            return (Offset == other.Offset)
+                && (SequenceNumber == other.SequenceNumber)
+                && (EnqueuedTime == other.EnqueuedTime)
+                && (LastReceivedTime == other.LastReceivedTime);
+        }
+
+        /// <summary>
+        ///   Determines whether the specified <see cref="System.Object" /> is equal to this instance.
+        /// </summary>
+        ///
+        /// <param name="obj">The <see cref="System.Object" /> to compare with this instance.</param>
+        ///
+        /// <returns><c>true</c> if the specified <see cref="System.Object" /> is equal to this instance; otherwise, <c>false</c>.</returns>
+        ///
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public override bool Equals(object obj) =>
```

Comment:
Thoughts on using `<inheritdoc />` for some of these doc comments where we're just implementing or overriding well known methods?

Reply:
Is that a universal concept in tooling these days?   When I first saw it, it was a niche thing that the MS Docs generator and, I think, Visual Studio used but it wasn't an official part of the XML doc comments spec.   I just looked, and I still don't see it called out in the [spec](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/documentation-comments).   Am I overlooking something?

That all said, assuming that I'm not up to date, I'd love to adopt it going forward, but don't want to take on a retrofit of the EH codebase;  I'd rather keep internal consistency and let GhostDoc populate the shim than apply the inherit haphazardly.   _(I have zero objections to someone else doing the work, though.  wink-wink-nudge-nudge.  :smile:)_

Reply:
Elsewhere on MSDN it is [documented](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/inheritdoc) and we do use it in other libraries in the tree.

> (I have zero objections to someone else doing the work, though. wink-wink-nudge-nudge. 😄)

SGTM - Maybe there will be a PR in your inbox on Monday.

Reply:
oooh... nice.   I learned me something.   I'll definitely start using that in the future.